net/quagga/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf

```diff
@@ -4,6 +4,14 @@
 ! Zebra configuration saved from vty
 !   2017/03/03 20:21:04
 !
+{% if helpers.exists('OPNsense.quagga.general') %}
+{%   if helpers.exists('OPNsense.quagga.general.enablelogfile') and OPNsense.quagga.general.enablelogfile == '1' %}
+log file /var/log/zebra.log {{ OPNsense.quagga.general.logfilelevel }}
```

Comment:
What about quagga.log? Since every daemon (also zebra) put their stuff in there.